frontend/src/App.tsx

```diff
@@ -16,9 +19,11 @@ export const App: React.FC = () => <>
             </Switch>
         </Root>
     </Router>
-</>;
+</RelayEnvironmentProvider>;
 
 
 const RealmPage: React.FC<RouteComponentProps<{ path?: string }>> = ({ match }) => (
-    <Realm path={match.params.path?.split("/") ?? []} />
+    <Suspense fallback="TODO">
+        <Realm path={match.params.path ?? ""} />
+    </Suspense>
```

Comment:
Another thing about this suspense solution:

- `Suspense` in this context is still unstable and react warns against using it stably already. Not happy about that.
- This now replaces the whole `Realm` component with some "loading thingy". But I guess we'd rather solve this within `Realm`. For example, the Breadcrumbs can already be rendered as we already have all the information locally. Lots of other layout stuff can also be rendered. 

I'd be fine with merging it like is, but yeah.

Reply:
Where do we have the information to render the breadcrumbs without making a request?

But sure, in general we can talk about factoring this differently.

I also just created #99 to track discussions about this.

Reply:
Ok then let's not do this in this PR. But I really want that, as in: use all the data that is already there.